validator/modules/llm_judge/__init__.py

```
@@ -0,0 +1,267 @@
+            completion = self.client.chat.completions.create(**params)
+            return completion.choices[0].message.content
+        except Exception as e:
+            raise LLMJudgeException(f"API call failed: {e}")
+
+    def _load_jsonl_conversations(self, data_url: str) -> List[Dict[str, Any]]:
+
+        response = requests.get(data_url)
+        response.raise_for_status()
+
+        conversations = []
+        for line_num, line in enumerate(response.text.strip().split("\n"), 1):
```

Ruff (0.12.2)

Loop control variable `line_num` not used within loop body

Rename unused `line_num` to `_line_num`

(B007)